Source/WEB_ROOT/admin/asset/js/addHistory.js

```diff
@@ -0,0 +1,22 @@
+function addRecordToHistoryDB(assetid,note)

```

Comment:
Great very readable method name and variable names. Everyone distinctly knows what you're doing, and what's required to do it.